src/docs/sdk/envelopes.mdx

```diff
@@ -400,14 +400,37 @@ update to an existing session for Release Health.
 
 *None*
 
-### UserReport
+### User Feedback
 
-Item type `"user_report"`. This Item contains a user report JSON payload.
+User Feedback was called User Report in the beginning. Therefore the envelope item type is `"user_report"`. 
+The item contains a user feedback / user report JSON payload:
+```json
+{"event_id":"9ec79c33ec9942ab8353589fcb2e04dc","email":"john@me.com","name":"John Me","comments":"It broke."}\n
+```
+
+**Attributes**
+
+`event_id`
+
+: **UUID String, required.** The identifier of the event or transaction.
+
+
+`email`
+
+: *String, recommended.* The email of the user.
+
+`name`
+
+: *String, recommended.* The name of the user.
+
+`comments`
+
+: *String, recommended.* Comments of the user about what happened. 
```

Comment:
Isn't this required? It seems that minimum is eventId, commments and (name or email)

Reply:
No, `eventId` plus any of the other is required. It's fine to send user feedback with only a comment for example.